narwhals/_polars/utils.py

```diff
@@ -94,10 +95,10 @@ def native_to_narwhals_dtype(dtype: Any, dtypes: DTypes) -> DType:
     return dtypes.Unknown()
 
 
-def narwhals_to_native_dtype(dtype: DType | type[DType], dtypes: DTypes) -> Any:
-    if (pl := get_polars()) is not None and isinstance(
-        dtype, (pl.DataType, pl.DataType.__class__)
-    ):
+def narwhals_to_native_dtype(dtype: DType | type[DType], dtypes: DTypes) -> pl.DataType:
+    import polars as pl  # ignore-banned-import()
+
+    if isinstance(dtype, (pl.DataType, pl.DataType.__class__)):  # type: ignore[arg-type]
```

Comment:
we're inside `_polars` here, so we shouldn't need `get_polars`, `polars` must already have been imported